README.md

```diff
@@ -1,74 +1,95 @@
-# FFC Pay Report Generator
+# Payment Hub Report Generator
 
-Template to support rapid delivery of microservices for FFC Platform. It contains the configuration needed to deploy a simple Hapi Node server to the Azure Kubernetes Platform.
+Microservice to periodically generate reports from Payment Hub event store.
```

Comment:
Should the period of time be included here or do you think the description is fine as is?

Reply:
Yes I think it would be useful.  I've added some detail further down in the repo to explain how it works in AKS, ie cron job every hour and that the Helm chart values control this.